cmake/Dependencies.aarch64-linux.cmake

```diff
@@ -95,3 +95,8 @@ get_link_libraries(PROTO_LIB_DEPS protobuf::libprotobuf)
 # libutf8_validity.a is a result of a generator expression, add it manually as it is
 # hard/impossible to learn its name during the configuration phase
 list(APPEND DALI_EXCLUDES ${PROTO_LIB_DEPS} libutf8_validity.a)
+
+##################################################################
+# pybind 11
+##################################################################
+set(PYBIND11_USE_CROSSCOMPILING ON)
```

Comment:
This helped fix the build errors, see https://pybind11.readthedocs.io/en/stable/changelog.html#version-2-13-0-june-25-2024